src/libraries/System.Linq/src/System/Linq/OrderBy.cs

```diff
@@ -7,12 +7,126 @@ namespace System.Linq
 {
     public static partial class Enumerable
     {
+        /// <summary>
+        /// Sorts the elements of a sequence in ascending order.
+        /// </summary>
+        /// <typeparam name="T">The type of the elements of <paramref name="source"/>.</typeparam>
+        /// <param name="source">A sequence of values to order.</param>
+        /// <returns>An <see cref="IOrderedEnumerable{TElement}"/> whose elements are sorted.</returns>
```

Comment:
Should there be noted
* in which order
* by what it is sorted

?

Reply:
> in which order

https://github.com/dotnet/runtime/blob/ea53a6c512309e8409ad70290f30a401ca3947e3/src/libraries/System.Linq/src/System/Linq/OrderBy.cs#L11

> by what it is sorted

It is ordered according to itself. I think this should result from the context instead of explicitly writing this here.